decisions/2019/05/25-war-docker-schemes.md

```diff
@@ -4,28 +4,31 @@
 
 #### Stable
 
-URL structure: `/dhis2-stable-{version}.war`
+URL structure: `{version}/dhis2-stable-{version}.war`
```

Comment:
Was wondering if we should be consistent with the others? I think it would look nicer:

`{version}/stable/dhis2-stable-${patch-version}`

e.g.

```
https://releases.dhis2.org/2.32/stable/dhis2-stable-2.32.0.war
https://releases.dhis2.org/2.32/canary/dhis2-canary-2.32.war
https://releases.dhis2.org/2.32/dev/dhis2-dev-2.32.war
```

Reply:
OK, I see now that is what you meant with the comment: https://github.com/dhis2/notes/pull/84/files#issuecomment-561519269